Firelocks can be opened by hand in reasonable time

moxian opened the pull request:
## What Does This PR Do
Reduces the time it takes to open firelock by hand (without a crowbar) from 30 seconds down to 5.

## Why It's Good For The Game
Atmos breaches are fairly regular on the station. Whenever one happens, firelocks, naturally, engage. However they often cut you off from the fire panel, so you effectively need a crowbar to get yourself to where you want to be (and often, to safety).
However, needing to carry a crowbar on you at all times exactly for the reason of "what if i get bombed?" is.. unexciting, especially if you play a role that shouldn't *need* to carry a crowbar IC'ly.

Now, turns out, that it *is* possible to open firelocks by hand, but it takes an excuriating 30 seconds, which is just waaay too long. Lowering that time to something - still long, but reasonable, like 5 seconds (or maybe 10 even) would eliminate the need to carry crowbar "just in case", and (arguably!) higher the RP standards a tiniest bit.

Please note that 5 seconds is still way way longer than the "instant" that you get when applying the crowbar, so crowbar absolutely does not become useless item.

## Images of changes
![20201108-025530-IhHankIt98](https://user-images.githubusercontent.com/7831163/98455848-f1fd6700-216d-11eb-9158-d919e93758cd.gif)


## Changelog
:cl:
tweak: Firelocks can be opened by hand in 5 seconds, down from 30
/:cl:

Mitchs98 commented:
I'll still carry a crowbar, and I don't think arguing for the PR in an IC sense is relevant. But this WILL be great for the rounds you spawn in to an arrivals breach and can't get a crowbar, or what have you.

Carthusia commented:
Perhaps 10-15 seconds, its still a long time, but nowhere near as bad.

You are trying to force open a hydraulically pressure sealed door.

There is also another PR right now to change the fireaxes opening speed to 7.5 seconds, i feel there is a bit of a conflict or inconsistency between this and that one, where opening a firelock with your hands would be far quicker than even using a fireaxe

moxian commented:
![dreamseeker-20201108-233948](https://user-images.githubusercontent.com/7831163/98487608-cc2e9b80-221b-11eb-89a5-0d682c5afd00.png)

> You are trying to force open a hydraulically pressure sealed door.
I agree it's silly.
I was originally inspired to do this by bee, where the firelocks mention that they have a mechanical lever on them, but was too lazy to do this properly (and they are in general closer to what we use regular airlocks for I think).
Anyhow, added the flavour.

Also removed the need for non-help intent to force it open. The "if it's a glass airlock, then tap on the glass instead" bit is kinda nice, but it took me reading the code, and turning my volume all way up to realize the sound is indeed there - it's that quiet.

> There is also another PR right now to change the fireaxes opening speed to 7.5 seconds, i feel there is a bit of a conflict or inconsistency between this and that one, where opening a firelock with your hands would be far quicker than even using a fireaxe.
Airlocks and firelocks are different entities with different behavior. Not saying this is how it should be, but that's how the game is right now.
Perhaps firelocks' motors are weaker than airlocks'? After all you can crowbar firelocks instantly, but airlocks not at all...

> Perhaps also add a timer to crowbaring the airlocks as well? 
I'm not brave enough to make that kind of PR :stuck_out_tongue:

moxian commented on code/game/machinery/doors/firedoor.dm:
```diff
@@ -298,7 +301,6 @@
 	opacity = 1
 	explosion_block = 2
 	assemblytype = /obj/structure/firelock_frame/heavy
-	can_force = FALSE
```
No longer makes flavour sense, given that we are no longer *literally* forcing the lock open by hand, but using the emergency fallback instead.

Carthusia commented:
Ooohh, i like the idea of it noting an emergency lever, makes MUCH more sense than trying to push the door up AND down at the same time with your bare hands

SabreML commented on code/game/machinery/doors/firedoor.dm:
```diff
@@ -22,10 +22,11 @@
-	var/can_force = TRUE
-	var/force_open_time = 300
+	/// How long does opening by hand take, in deciseconds.
+	var/manual_open_time = 50
```
```suggestion
	var/manual_open_time = 5 SECONDS
```

SabreML commented on code/game/machinery/doors/firedoor.dm:
```diff
@@ -89,20 +90,22 @@
-							 "<span class='warning'>You bang on \the [src].</span>")
-		playsound(get_turf(src), 'sound/effects/glassknock.ogg', 10, 1)
+	user.visible_message(
+		"<span class='notice'>[user] tries to open \the [src] manually.</span>",
+		"<span class='notice'>You operate the manual lever on \the [src].</span>")
```
You might not need the `\the`s here, `[src]` usually adds them automatically.

SabreML commented on code/game/machinery/doors/firedoor.dm:
```diff
@@ -89,20 +90,22 @@
+
+	if(do_after(user, manual_open_time, target = src))
+		user.visible_message(
+			"<span class='notice'>[user] opens \the [src].</span>",
+			"<span class='notice'>You open \the [src].</span>")
```
Same here
